tools/admin/requests/create-new.ts

```diff
@@ -194,6 +194,7 @@ export const INITIAL_ADDITIONAL_QUESTIONS: AdditionalInformationFormData = {
 // Initial data for payment details in application forms
 export const INITIAL_PAYMENT_DETAILS: PaymentInformationFormData = {
   paymentMethod: null,
+  processingFee: '26',
```

Comment:
Should this be process.env.PROCESSING_FEE like it is in lib/application/resolver.ts?

Reply:
Ideally, but react env variables need to be prefixed with `REACT_APP_` so I'd need to define a new one and set it on the Heroku instance. Let's go with this hardcode right now and I'll drop a ticket in the backlog to clean up.